Assets/Project/Scripts/Networks/Requests/HelloWorldRequest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using UnityEngine.Networking;
+
+namespace Project.Scripts.Networks.Requests
+{
+    public class HelloWorldRequest : GetServerRequest
+    {
+        public HelloWorldRequest()
+        {
+            ServerRequest = UnityWebRequest.Get(NetworkService.HOST);
+        }
+
+        public override void SetCache()
+        {
+            throw new System.NotImplementedException();
+        }
+
+        protected override object DeserializeResponse()
+        {
+            if (!IsRemoteDataValid())
+                return GetData_Local();
+
+            return ServerRequest.downloadHandler.text;
```

Comment:
```suggestion
            return !IsRemoteDataValid() ? GetData_Local() : ServerRequest.downloadHandler.text;
```

逆に読みづらくなるという気持ちもわかるから，こういう書き方もあるよってだけ伝える．任意で

Reply:
昔の僕はこう書いたんだろうけど、リーダブルコードを読んだらやっぱり分けた方が読みやすいと思った。

Reply:
理解！🙆‍♂️
リーダブルコードって特定の本とかドキュメントだっけ？

Reply:
[こちらの本](https://www.amazon.co.jp/リーダブルコード-―より良いコードを書くためのシンプルで実践的なテクニック-Theory-practice-Boswell/dp/4873115655/ref=sr_1_1?__mk_ja_JP=カタカナ&crid=L27WG39NC3EY&dchild=1&keywords=リーダブルコード&qid=1589299279&s=books&sprefix=リーダブル%2Caps%2C-1&sr=1-1)のことです

Reply:
あ，これね．オッケ
もし余裕があるタイミングあったら，wiki かなんかにこれは共有したいというポリシーを書いてみて．